### DISCUSSION TITLE
Unable to Verify Domain

### DISCUSSION THREAD

**Discussion by tonyoconnell** [swellstores/community#118] [Help] [answered]
Hello. I am writing this here because this is the only place I can contact support without paying $300 a month, am I right? I am unable to verify my domain from storename.swell.store/admin/store?tab=domains . I am using Cloudflare for my DNS. I have the CNAME setup correctly. I have the proxy turned off. It's been past 100 hours since it's been setup and for sure the DNS has propagated. The website works but the domain can't be verified. What should I do?

**Comment by cristianstanescu** [comment, marked as answer]
Hi @tonyoconnell. Sorry to hear that setting up the domain does not work for you.

The instructions for setting up a custom domain are available in our developer docs: https://developers.swell.is/storefronts/custom-domains-and-ssl

If you feel you’ve reached a bug, please share your account name and detailed description of the issue with [support@swell.is](mailto:support@swell.is) so we can look into it further.
